### ISSUE TITLE
Windows line endings

### ISSUE THREAD

**Issue by lizardstarks** [donkirkby/live-py-plugin#181]
Testing this in pycharm in Windows, I encountered multiple unit test failures due to line endings. The line endings also interfere with turtle graphics.

```
sys.stdout.write('a\nb\n')
 != print('a\nb')


Expected :print('a\nb')

Actual   :sys.stdout.write('a\nb\n')
```

**Comment by salvor7** [comment]
I'll have a look at fixing this during the sprint.

**Comment by donkirkby** [comment]
Fixed in pull request #182. Commit 52065bdb85e696c089f212e0a987437715ee90c6 got rebased, because of some merge problems.
